releases/release-1.22/release-notes/maps/pr-102133-map.yaml

```diff
@@ -0,0 +1,6 @@
+pr: 102133
+releasenote:
+  text: '`kubeadm`: during "kubeadm init/join/upgrade", always default the "cgroupDriver"
```

Comment:
Again about the "" and `` Also you can place the link provided in a markdown style link